src/bootstrap/config.rs

```diff
@@ -308,7 +308,7 @@ pub struct Target {
 impl Target {
     pub fn from_triple(triple: &str) -> Self {
         let mut target: Self = Default::default();
-        if triple.contains("-none") || triple.contains("nvptx") {
+        if triple.contains("-none") || triple.contains("nvptx") || triple.contains("switch") {
```

Comment:
Off topic - could this be turned into a `no_std` flag in the target config file?

Reply:
The linker has the same issue with a `FIXME` for pulling the info from the target config file. I'm guessing that `bootstrap` doesn't have a way to access the actual target definition.

Reply:
In this PR, though, it's just temporary until std *is* implemented on Switch.